cmd/earth/main.go

```diff
@@ -1326,9 +1327,19 @@ func (app *earthApp) actionBuild(c *cli.Context) error {
 
 	sc := secretsclient.NewClient(app.apiServer, app.sshAuthSock, app.publicKey)
 
+	cacheLocalDir, err := ioutil.TempDir("/tmp", "earthly-cache")
```

Comment:
if the first arg to TempDir is `""`, then it'll use the OS's default tmp dir location (which will most likely be /tmp, but you never know...)